packages/handlersjs-http/lib/handlers/json-store-handler.spec.ts

```diff
@@ -0,0 +1,142 @@
+import { MemoryStore } from '@digita-ai/handlersjs-core';
+import { HttpHandlerContext } from '../models/http-handler-context';
+import { HttpHandlerResponse } from '../models/http-handler-response';
+import { JsonStoreHandler } from './json-store-handler';
+
+describe('JsonStoreHandler', () => {
+
+  interface StoreInterface {
+    data: string[];
+  }
+
+  const inputData = [ 'abc', 'defghij', '123' ];
+  const inputData2 = [ ... inputData, '456' ];
```

Comment:
What will this 'second data' be used for? A name which simply adds an index is not very clear...